### PR TITLE
Update Analytics Android and iOS bundled SDKs

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/ACPAnalyticsAndroidUnitTests/ACPAnalyticsAndroidUnitTests.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ public void Setup()
         public void GetACPAnalyticsExtensionVersion_Returns_CorrectVersion()
         {
             // verify
-            Assert.That(ACPAnalytics.ExtensionVersion(), Is.EqualTo("1.2.6"));
+            Assert.That(ACPAnalytics.ExtensionVersion(), Is.EqualTo("1.2.9"));
```
Comment (rymorale) [inline src/ACPAnalyticsAndroidUnitTests/ACPAnalyticsAndroidUnitTests.cs:45]: should the latest 1.2.10 analytics release be used?

Comment (cacheung) [reply src/ACPAnalyticsAndroidUnitTests/ACPAnalyticsAndroidUnitTests.cs:45]: oh yeah, it was released last night.  It fixed an assurance integration issue which no debug flag showing up in some events.

**Review by rymorale — APPROVED** [review]
looks good to me overall, just wondering if the latest android analytics release should be used instead

File: src/ACPAnalyticsAndroidUnitTests/ACPAnalyticsAndroidUnitTests.cs (hunk 1 above)
Comment (spoorthipujariadobe) [reply src/ACPAnalyticsAndroidUnitTests/ACPAnalyticsAndroidUnitTests.cs:45]: Will updated, created this PR yesterday so might have missed it

**Review by cacheung — COMMENTED** [review]
Do we need to update the version in these two files?
https://github.com/spoorthipujariadobe/xamarin-acpanalytics/blob/update_analytics_to_latest/src/Adobe.ACPAnalytics.tvOS/Properties/AssemblyInfo.cs#L28

https://github.com/spoorthipujariadobe/xamarin-acpanalytics/blob/update_analytics_to_latest/src/Adobe.ACPAnalytics.iOS/Properties/AssemblyInfo.cs#L40

**Comment by spoorthipujariadobe** [comment]
> Do we need to update the version in these two files? https://github.com/spoorthipujariadobe/xamarin-acpanalytics/blob/update_analytics_to_latest/src/Adobe.ACPAnalytics.tvOS/Properties/AssemblyInfo.cs#L28
> 
> https://github.com/spoorthipujariadobe/xamarin-acpanalytics/blob/update_analytics_to_latest/src/Adobe.ACPAnalytics.iOS/Properties/AssemblyInfo.cs#L40

I didn't update the tvOS SDK as the newer tvOS Core SDK was giving errors in Xamarin. Since no customers are using it yet, we might just leave it as if till we get a request.

**Comment by cacheung** [comment]
How about this file?
https://github.com/spoorthipujariadobe/xamarin-acpanalytics/blob/update_analytics_to_latest/src/Adobe.ACPAnalytics.iOS/Properties/AssemblyInfo.cs#L40

**Comment by spoorthipujariadobe** [comment]
> How about this file? https://github.com/spoorthipujariadobe/xamarin-acpanalytics/blob/update_analytics_to_latest/src/Adobe.ACPAnalytics.iOS/Properties/AssemblyInfo.cs#L40

Will update that, thank you

**Comment by spoorthipujariadobe** [comment]
> > Do we need to update the version in these two files? https://github.com/spoorthipujariadobe/xamarin-acpanalytics/blob/update_analytics_to_latest/src/Adobe.ACPAnalytics.tvOS/Properties/AssemblyInfo.cs#L28
> > https://github.com/spoorthipujariadobe/xamarin-acpanalytics/blob/update_analytics_to_latest/src/Adobe.ACPAnalytics.iOS/Properties/AssemblyInfo.cs#L40
> 
> I didn't update the tvOS SDK as the newer tvOS Core SDK was giving errors in Xamarin. Since no customers are using it yet, we might just leave it as if till we get a request.

Was able to fix tvOS issues by clearing local cache. Updated all assembly files as well. Thank you!

**Review by cacheung — APPROVED** [review]

**Pull request merged**